packages/opentelemetry-plugin-xml-http-request/src/enums/AttributeNames.ts

```diff
@@ -0,0 +1,27 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export enum AttributeNames {
+  COMPONENT = 'component',
+  HTTP_HOSTNAME = 'http.hostname',
+  HTTP_METHOD = 'http.method',
+  HTTP_PATH = 'http.path',
+  HTTP_URL = 'http.url',
+  HTTP_STATUS_CODE = 'http.status_code',
+  HTTP_STATUS_TEXT = 'http.status_text',
+  // NOT ON OFFICIAL SPEC
```

Comment:
Are the other ones official spec for OpenTelemetry? If so, could you link to it in a JSDoc comment?

Reply:
https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-http.md#common-attributes, spec has changed. Need to sync as well as http/https plugins.